test/distributed/tensor/test_random_ops.py

```diff
@@ -130,6 +131,7 @@ def test_meta_tensor_init(self):
                 )
 
         # Test 2: disable the distribute region for RNG
+        torch.get_device_module(self.device_type).manual_seed(self.rank)
```

Comment:
This change feels suspicious because it adds a new set seed where there previously wasn't one. Also it missed changing a cuda-specific seed setting in the same test further above.